packages/integration-matomo/src/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import {
+  getForegroundColor,
+  IntegrationConfig,
+  createIconComponentFromSimpleIconsSvgPath,
+  useIntegration,
+  IntegrationConfigOptions,
+} from '@techboi/consent-manager'
+
+import Matomo from 'simple-icons/icons/matomo'
+
+declare global {
+  interface Window {
+    _paq: any
```

Comment:
Would this work as well?

```suggestion
    _paq?: unknown[]
```

Reply:
After matomo initialized this is not a regular array anymore, AFAIK. We might check

Reply:
Does not complain 🙃